tests/php/Model/RedirectorPageTest.php

```diff
@@ -117,4 +124,25 @@ public function testNoJSLinksAllowed()
         $page->write();
         $this->assertEmpty($page->ExternalURL);
     }
+
+    public function testOnMissingLinkResponse()
+    {
+        $page = $this->objFromFixture(RedirectorPage::class, 'goodinternal');
+        $this->assertEquals('/redirection-dest/', $page->Link());
+        $response = $this->get($page->Link());
```

Comment:
I note that other tests use `Director::makeRelative()` with this call, you may want to copy that